Created NavBar, SideBar and CategoryDashboard component

abhishek2x opened the pull request:
The components are temporarily placed in the manager.tsx file.

upsurge0 reviewed (changes requested):
### List of Changes:
- Made css changes and added components to /tickets route.
- Installed @svgr/webpack to use svg's as a component.
- Removed /static folder and moved icons to /public.

### Please make the following changes:
 - Add next/link to NavBar .boxItems div.
 - Use .selected CSS class for currently select route. [(class already created)](https://github.com/abhishek2x/tickvit-next/blob/6fcd8515741134f416bf5ed691bbf86b99d039c6/components/NavBar/Navbar.tsx#L15e)

abhishek2x commented:
> ### List of Changes:
> * Made css changes and added components to /tickets route.
> * Installed @svgr/webpack to use svg's as a component.
> * Removed /static folder and moved icons to /public.
> 
> ### Please make the following changes:
> * Add next/link to NavBar .boxItems div.
> * Use .selected CSS class for currently select route. [(class already created)](https://github.com/abhishek2x/tickvit-next/blob/6fcd8515741134f416bf5ed691bbf86b99d039c6/components/NavBar/Navbar.tsx#L15e)

### Regarding the requested changes:
1. fixed
2. I'm extracting current URL route using ( { withRouter } from 'next/router') and storing it is the "selected" state. Based on the value of the selected state, we can make that icon active. 

LMK if this approach is fine, or if there is some better way to go about it.

upsurge0 commented:
> > ### List of Changes:
> > 
> > * Made css changes and added components to /tickets route.
> > * Installed @svgr/webpack to use svg's as a component.
> > * Removed /static folder and moved icons to /public.
> > 
> > ### Please make the following changes:
> > 
> > * Add next/link to NavBar .boxItems div.
> > * Use .selected CSS class for currently select route. [(class already created)](https://github.com/abhishek2x/tickvit-next/blob/6fcd8515741134f416bf5ed691bbf86b99d039c6/components/NavBar/Navbar.tsx#L15e)
> 
> ### Regarding the requested changes:
> 1. fixed
> 2. I'm extracting current URL route using ( { withRouter } from 'next/router') and storing it is the "selected" state. Based on the value of the selected state, we can make that icon active.
> 
> LMK if this approach is fine, or if there is some better way to go about it.

Yes, you can go ahead with this approach.
